tests/v3/test_store/test_remote.py

```diff
@@ -113,8 +107,8 @@ def store_kwargs(self, request) -> dict[str, str | bool]:
         raise AssertionError
 
     @pytest.fixture(scope="function")
-    def store(self, store_kwargs: dict[str, str | bool]) -> RemoteStore:
-        url = store_kwargs["url"]
+    async def store(self, store_kwargs: dict[str, str | bool | UPath]) -> RemoteStore:
```

Comment:
This isn't actually async

Reply:
correct, but the class we are inheriting from defines this as an async method